src/webgl/p5.RendererGL.Retained.js

```diff
@@ -50,32 +69,41 @@ p5.RendererGL.prototype.createBuffers = function(gId, obj) {
     shader = this.setShader(this._getColorShader());
   }
 
-  // allocate space for vertex positions
-  var data = new Float32Array(vToNArray(obj.vertices));
+  /****THIS IS HOW VERTICES WERE DRAWN BEFORE****/
+
+  // // allocate space for vertex positions
+  // var data = new Float32Array(vToNArray(obj.vertices));
+  // shader.enableAttrib(shader.attributes.aPosition.location,
+  //   3, gl.FLOAT, false, 0, 0);
+  // gl.bindBuffer(gl.ARRAY_BUFFER, this.gHash[gId].vertexBuffer);
+  // gl.bufferData(gl.ARRAY_BUFFER, data, gl.STATIC_DRAW);
+
+  var data = new Float32Array(flatten(flatten(obj.lineVertices)));
   shader.enableAttrib(shader.attributes.aPosition.location,
     3, gl.FLOAT, false, 0, 0);
-  gl.bindBuffer(gl.ARRAY_BUFFER, this.gHash[gId].vertexBuffer);
+  gl.bindBuffer(gl.ARRAY_BUFFER, this.gHash[gId].lineVertexBuffer);
   gl.bufferData(gl.ARRAY_BUFFER, data, gl.STATIC_DRAW);
```

Comment:
Cool, I think it makes sense that these line vertices would be pushed to the aPosition attribute. It would make writing custom shaders easier by requiring the user to know less about our implementation, which is a good thing.